### PR TITLE
admin-dashboard: overhaul

### PR DESCRIPTION
**Pull request by mattiascockburn**
- incorporate some custom styles for the default dashboard
- use a stripped down image and busybox httpd
- add a more descriptive README

### CODE REVIEW

**Comment by victorgetz** [comment]
if i apply it i get this error:

one or more objects failed to apply, reason: ConfigMap "admin-dashboard" is invalid: metadata.annotations: Too long: must have at most 262144 bytes,Deployment.apps "admin-dashboard" is invalid:

**Comment by victorgetz** [comment]
I would make the index.html configurable because the links needs to flexible

**Comment by mattiascockburn** [comment]
> I would make the index.html configurable because the links needs to flexible

The website is overridable through an entry in the values file. Only if that value is empty
the template will inject the default HTML file.

**Comment by victorgetz** [comment]
rejected for chatgpt solution

File: charts/iits-admin-dashboard/values.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+resources:
+  requests:
+    memory: "4Mi"
+    cpu: "1m"
+  limits:
+    memory: "8Mi"
+    cpu: "10m"
+# You can add here a custom HTML website
+#website: |-
+
```
Comment (mattiascockburn) [inline charts/iits-admin-dashboard/values.yaml:20]: That's a bunch of newlines ;-)

**Review by mattiascockburn — COMMENTED** [review]
LGTM

**Pull request merged**